pages/09.contacts/04.manage-fields/docs.en.md

```diff
@@ -44,3 +44,13 @@ You can create additional custom fields and define the data type you want that f
 
 [contact monitoring]: </contacts/manage-contacts/contact-monitoring>
 [variables]: </setup/variables>
+
+### Creating custom fields via a command
+
+Each new custom field for contact or a company is adding new column to the database. This operation gets slower with more data. It also locks the table so no changes can come in for tens of minutes in some cases. It will also time out the HTTP request so the UI will report the column exist but contact/company updates will actually fail because the column is missing. There is a way around when you set a background processing of field creation. 
+
+There is now an option you can set in your `app/config/local.php` file: `'create_custom_field_in_background' => true,`. If this is set then only the field metadata will be created so you will be able to see the new custom field in the list of custom fields. But it will be unpublished until a command `bin/console mautic:custom-field:create-column` runs. This command will create the actual column in the table and publishes the field metadata.
+
+With this the HTTP request timeout is fixed because the long running SQL query that is creating the new table column is handled in a background task.
```

Comment:
```suggestion
With this configuration enabled, the HTTP request timeout is prevented because the long running SQL query that is creating the new table column is handled in a background task.
```